### PR TITLE
BC-4323 add ROOT_URL_REDIRECT to ansible/group_vars/all/config.yml

### PR DESCRIPTION
**Pull request by Loki-Afro**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-4323

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -798,4 +798,8 @@ configuration_all:
   PROMETHEUS_METRICS_COLLECT_METRICS_ROUTE_METRICS:
     server: true
     client: false
-    nuxtclient: false
\ No newline at end of file
+    nuxtclient: false
+  ROOT_URL_REDIRECT:
+    server: false
```
Comment (CeEv) [inline ansible/group_vars/all/config.yml:803]: No value exists for it? Not in one special instance?

File: ansible/group_vars/all/config.yml [hunk 2]
```diff
@@ -798,4 +798,8 @@ configuration_all:
...
+    client: true
+    nuxtclient: false
```
Comment (mamutmk5) [inline ansible/group_vars/all/config.yml:805]: set the value to "" as default to make sure that there is no value for it
```suggestion
  ROOT_URL_REDIRECT:
    value: ""
    server: false
    client: true
    nuxtclient: false
```

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/config.yml (hunk 2 above)
Comment (mamutmk5) [reply ansible/group_vars/all/config.yml:805]: Line of code where it use at the legathy client https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/app.js#L208-L213

File: ansible/group_vars/all/config.yml (hunk 1 above)
Comment (Loki-Afro) [reply ansible/group_vars/all/config.yml:804]: took me a few attempts to get what you meant :) the values are already there but this block was missing for whatever reason

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**